week2/exercise-template/src/backend/data/movies.json

```diff
@@ -1328,7 +1328,7 @@
         "id": 166
     },
     {
-        "title": "A Nightmare on Elm Street 4: The Dream Master",
+        "title": "A Nightmare on Elm Street 4: The Dream main",
```

Comment:
I think this is a movie title :) 

Should we replace the whole title @martamatos ?